src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx

```diff
@@ -110,6 +114,6 @@ export default function createQueryBuilder(initial = {}, organization) {
 
   // Get all columns, including tags
   function getColumns() {
-    return [...COLUMNS, ...tags.map(tag => ({name: tag, type: 'string'}))];
+    return [...COLUMNS, ...tags];
```

Comment:
Would it be better to make this an async function, otherwise seems easy for someone to call `getColumns` without calling `load` first.